pkg/network/ndmz/dualstack.go

```diff
@@ -25,8 +25,9 @@ import (
 
 // DualStack implement DMZ interface using dual stack ipv4/ipv6
 type DualStack struct {
-	nodeID     string
-	ipv6Master string
+	nodeID          string
+	ipv6Master      string
+	backgroundProbe *dhcp.BackgroundProbe
```

Comment:
I don't think you need to store this here, because it's a stateless object. it means you still can create another one with the same "interface" name and use it any time.